docs/release-notes/release-notes-1.6.0-rc1.1.md

```diff
@@ -0,0 +1,925 @@
+lowered.
+
+These changes, in conjunction with other related changes, mean that nodes are
+better about orphan transaction management and thus missing ancestors will
+typically either be broadcast or mined fairly quickly resulting in fewer overall
+orphans and smaller actual run-time orphan pools.
+
+### Ticket Exhaustion Prevention
+
+Mining templates that would lead to the chain becoming unrecoverable due to
+inevitable ticket exhaustion will no longer be generated.
+
+This is primarily aimed at the testing networks, but it could also theoretically
+affect the main network in some far future if the demand for tickets were to
+ever dry up for some unforeseen reason.
+
+### New Initial State Protocol Messages (`getinitstate`/`initstate`)
+
+This release introduces a pair of peer-to-peer protocol messages named
+`getinitstate` and `initstate` which support querying one or more pieces of
+information that are useful to acquire when a node first connects in a
+consolidated fashion.
+
+Some examples of the aforementioned information are the mining state as of the
+current tip block and, with the introduction of the decentralized treasury, any
+outstanding treasury spend transactions that are being voted on.
+
+### Mining State Protocol Messages Deprecated  (`getminings`/`minings`)
+
+Due to the addition of the previously-described initial state peer-to-peer
+protocol messages, the `getminings` and `minings` protocol messages are now
+deprecated.  Use the new `getinitstate` and `initstate` messages with the
+`headblocks` and `headblockvotes` state types instead.
+
+### RPC Server Changes
+
+The RPC server version as of this release is 6.1.2.
+
+#### New Treasury Balance Query RPC (`gettreasurybalance`)
+
+A new RPC named `gettreasurybalance` is now available to query the current
+balance of the decentralized treasury.  Please note that this requires the
+decentralized treasury vote to pass and become active, so it will return an
+appropriate error indicating the decentralized treasury is inactive until that
+time.
+
+See the
+[gettreasurybalance JSON-RPC API Documentation](https://github.com/decred/dcrd/blob/master/docs/json_rpc_api.mediawiki#gettreasurybalance)
+for API details.
+
+#### New Treasury Spend Vote Query RPC (`gettreasuryspendvotes`)
+
+A new RPC named `gettreasuryspendvotes` is now available to query vote
+information about one or more treasury spend transactions.  Please note that
+this requires the decentralized treasury vote to pass and become active to
+produce a meaningful result since treasury spend transactions are invalid until
+that time.
+
+See the
+[gettreasuryspendvotes JSON-RPC API Documentation](https://github.com/decred/dcrd/blob/master/docs/json_rpc_api.mediawiki#gettreasuryspendvotes)
+for API details.
+
+#### New Force Mining Template Regeneration RPC (`regentemplate`)
+
+A new RPC named `regentemplate` is now available which can be used to force the
+current background block template to be regenerated.
```

Comment:
If dcrpool takes advantage of this I would mention

Reply:
It doesn't need to.  It uses `notifywork`.

Reply:
Ok found the motivation for `regentemplate` in https://github.com/decred/dcrd/pull/1979